local-cli/runIOS/__tests__/findMatchingSimulator-test.js

```diff
@@ -0,0 +1,195 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+jest.dontMock('../findMatchingSimulator');
+
+const findMatchingSimulator = require('../findMatchingSimulator');
+
+describe('findMatchingSimulator', () => {
+  it('should find simulator', () => {
+    expect(findMatchingSimulator({
+        "devices": {
```

Comment:
Personally I find it very hard to read this test file. I believe it will make it easier if you use same inputs for each test – currently the search token ('iPhone 6') and expected result are deeply burrowed inside the test function, and the IDs are very random.